### DISCUSSION TITLE
Kitty window cannot be moved between macOS desktops

### DISCUSSION THREAD

**Discussion by wagaaa** [kovidgoyal/kitty#6124] [Q&A] [answered]
![ezgif-2-91f628d634](https://user-images.githubusercontent.com/13414470/226564093-9e9b4fb3-c926-4f78-90b6-538d30e6c74c.gif)
 
My kitty window seemed to be pinned on the top of all apps. As I  switch to another desktop, kitty window always be there.

<img width="280" alt="image" src="https://user-images.githubusercontent.com/13414470/226561110-8e993c7d-cb00-49de-8771-cb56a8c87273.png">

And I can't move it to another desktop as the cursor showed🚫. 

How can I fix it?

**Comment by kovidgoyal** [comment]
Turn off secure text input .

On Tue, Mar 21, 2023 at 02:23:39AM -0700, wagaaa wrote:
> ![ezgif-2-91f628d634](https://user-images.githubusercontent.com/13414470/226564093-9e9b4fb3-c926-4f78-90b6-538d30e6c74c.gif)
>  
> My kitty window seemed to be pinned on the top of all apps. As I  switch to another desktop, kitty window always be there.
> 
> <img width="280" alt="image" src="https://user-images.githubusercontent.com/13414470/226561110-8e993c7d-cb00-49de-8771-cb56a8c87273.png">
> 
> And I can't move it to another desktop as the cursor showed🚫. 
> 
> How can I fix it? 
> 
> 
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/6124
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Reply by wagaaa** [reply]
the "Secure Keyboard Entry" option in menu has already disabled,

<img width="231" alt="image" src="https://user-images.githubusercontent.com/13414470/226567690-d2ca5346-474f-40d2-9db1-077b85b37860.png">

and I tried redisable it, nothing worked😢

**Comment by page-down** [comment, marked as answer]
@wagaaa

> My kitty window seemed to be pinned on the top ...
> And I can't move it ...

That's because you used some third-party software, but you forgot about it or didn't know that it had modified the attributes of the kitty window.

> How can I fix it?

Exclude kitty from your third-party window management tools. Don't pin it in all spaces.

Avoid kitty OS window's collection behavior being set to `NSWindowCollectionBehaviorCanJoinAllSpaces`.
